packages/calypso-e2e/src/lib/utils/get-test-account.ts

```diff
@@ -0,0 +1,70 @@
+type Env = 'edge' | 'production';
+
+type SiteType = 'simple' | 'atomic';
```

Comment:
The presence of this here is bit of a problem, since an account can have multiple sites of which some can be `atomic` and some can be `simple`. 

With that said, I don't have a good solution to this, unless we want to start abstracting sites - but that seems rather excessive, so as long as we stay within the bounds of test accounts this assumption should hold.

Reply:
Ah, that's a good point! I think we have been following this pattern of one site per account right? Do you know of any editor tests that actually select a different site after login from the top of your head?